sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/Customizations/Models/Quaternion.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.MixedReality.ObjectAnchors.Models
+{
+    /// <summary> The Quaternion. </summary>
+    internal partial class Quaternion : IEquatable<Quaternion>
```

Comment:
Should this be implicitly convertible to and from a `System.Numerics.Quaternion`?

Personally, I'd find the existence of this type obnoxious, but the implicit conversion would make it more tolerable.

Reply:
Trying to make sure the wrapper class and the actual numerics class reference the same underlying data fields. It's internal and inevitable these classes get generated unfortunately, but I'd like to treat both objects like the same reference and not make getters which spit out a new object every time that has no impact on the values of the field it was derived from.

Reply:
I missed the fact that these are internal. I'll have to revisit how all this works as I seem to have missed something.